12-hooks-in-react/src/components/Route.js

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
+
+const Route = ({ path, children }) => {
+  //State to update Route Component
+  const [currentPath, setCurrentPath] = useState(window.location.pathname);
+
+  useEffect(() => {
+    const onLocationChange = () => {
+      setCurrentPath(window.location.pathname);
```

Comment:
Is this part of the lesson? I'm asking cause if it's not you can use the react router